src/main/log/logManager.ts

```diff
@@ -30,7 +30,7 @@ export class LogManager implements ExtensionComponent {
      * @param shouldToast - True iff toast should be shown
      */
     public logError(error: Error, shouldToast: boolean) {
-        this.logMessage(error.name, 'ERR');
+        this.logMessage(error.name || error.message, 'ERR');
```

Comment:
This is not a boolean type, you can logMessage accept string value only.
this syntax basically checks if error.name evaluates to false. If it does, it "returns" "error.message", otherwise it returns "error.name".